packages/next/src/server/app-render/entry-base.ts

```diff
@@ -29,8 +29,11 @@ import * as serverHooks from '../../client/components/hooks-server-context'
 import { HTTPAccessFallbackBoundary } from '../../client/components/http-access-fallback/error-boundary'
 import { createMetadataComponents } from '../../lib/metadata/metadata'
 import { patchFetch as _patchFetch } from '../lib/patch-fetch'
-// not being used but needs to be included in the client manifest for /_not-found
+// Not being directly used but should be included in the client manifest for /_not-found
```

Comment:
Is this a problem that we include it even if the user has a custom one?

Reply:
We still have the default global-error on the very top-level in case when user's global-error throws